src/Context/PageObjectContextTrait.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace PaulGibbs\WordpressBehatExtension\Context;
+
+use Behat\Behat\Context\Context;
+use SensioLabs\Behat\PageObjectExtension\PageObject\Factory as PageObjectFactory;
+use SensioLabs\Behat\PageObjectExtension\PageObject\Page;
+use SensioLabs\Behat\PageObjectExtension\PageObject\Element;
+
+trait PageObjectContextTrait
+{
+    /**
+     * @var PageObjectFactory
+     */
+    private $pageObjectFactory = null;
```

Comment:
Please use _ separated variable names for class properties, like existing classes.